.github/workflows/ci.yml

```diff
@@ -214,6 +214,57 @@ jobs:
 
       - run: go -C ./_tools run ./cmd/checkmodpaths $PWD
 
+  baselines:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          submodules: true
+      - uses: actions/setup-node@cdca7365b2dadb8aad0a33bc7601856ffabcc48e # v4.3.0
+      - uses: ./.github/actions/setup-go
+        with:
+          cache-name: baselines
+
+      - run: npm ci
+
+      - name: Remove all baselines
+        run: rm -rf testdata/baselines/reference
+
+      - name: Run tests
+        run: npx hereby test &> /dev/null || exit 0
+
+      - name: Accept baselines
+        run: |
+          npx hereby baseline-accept
+          # TODO: remove this once trace baselines are enabled
+          git restore 'testdata/baselines/reference/module/resolver/*.trace.json'
```

Comment:
These are the only baselines not recreated by the tests.

Reply:
I'm not sure how close I am to being able to do this; we could just delete them and then I can reference this PR to bring them back.